myutils.py

```diff
@@ -86,6 +86,27 @@ def getVendrediChillCategory(self):
     def getPotTresConfineCategory(self):
         return discord.utils.get(self.guild.categories, name="Pot Très Confiné")
 
+    def getLanOrgaCategory(self):
+        return discord.utils.get(self.guild.categories, name="Orga LAN")
+
+    def getViewersCategory(self):
+        return discord.utils.get(self.guild.categories, name="Viewers")
+
+    def getLOLCategory(self):
+        return discord.utils.get(self.guild.categories, name="League of Legends")
+
+    def getRlCategory(self):
+        return discord.utils.get(self.guild.categories, name="Rocket League")
+
+    def getCsgoCategory(self):
+        return discord.utils.get(self.guild.categories, name="CS:GO")
+
+    def getMinecraftCategory(self):
+        return discord.utils.get(self.guild.categories, name="Minecraft")
+
+    def getAoE2Category(self):
+        return discord.utils.get(self.guild.categories, name="AOE2")
```

Comment:
pareil ici, ces fonctions ne sont utilisées une seule fois, il vaudrait mieux une fonction générique à laquelle tu donnes un string qui est le nom de la catégorie, comme ça tu itères sur le tableau de jeux pour détruire les channels
ça évite de polluer utils avec des fonctions identiques et pas réutilisables

Reply:
c'est pas faux aussi 😉